examples/external-api-request/sign_manifest/sign.py

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env python3
+import argparse
+import base64
+import getpass
+from OpenSSL import crypto
+from pathlib import Path
+import yaml
+import json
+
+OUTPUT_YAML = "yaml"
+OUTPUT_JSON = "json"
+
+
+def build_parser():
+    parser = argparse.ArgumentParser(description="Sign a Golem manifest file.")
+    parser.add_argument(
+        "-k",
+        "--key",
+        dest="key",
+        required=True,
+        type=Path,
+        help="JSON file containing the manifest to be signed.",
+    )
+    parser.add_argument(
+        "-m",
+        "-manifest",
+        dest="manifest",
+        type=Path,
+        required=True,
+        help="JSON file containing the manifest to be signed.",
+    )
+    parser.add_argument(
+        "-c",
+        "-cert",
+        dest="cert",
+        type=Path,
+        required=False,
+        help="Optional certificate, to be included in the output",
+    )
+    parser.add_argument(
+        "-f",
+        "-format",
```

Comment:
`-f -format ` arg is a bit unnecessary and makes example more complicated.
In general having a uny utility code in `yapapi` examples feels like admitting to a defeat when it comes to usability.